modules/machine-config-node-drain.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * machine-configuration/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-config-node-drain_{context}"]
+= Understanding Machine Config Operator node drain behavior
```

Comment:
Sometimes I get confused when it's appropriate to use an article with MCO. Is it here? 

```suggestion
= Understanding the Machine Config Operator node drain behavior
```